package.json

```diff
@@ -11,6 +11,7 @@
 		"editor"
 	],
 	"dependencies": {
+		"@babel/plugin-transform-react-jsx": "7.3.0",
```

Comment:
wait, isn't this already included in our babel preset?

Reply:
Correct, it is:
https://github.com/WordPress/gutenberg/blob/master/packages/babel-preset-default/index.js#L18-L20

Reply:
Yeah, that was a shortcut while I figured out how to use `@wordpress/babel-preset-default` here. e2f45efa52deda2db8f073c28cd99cc1c39874ca and 8a4496aca692471adb932d44ca0641e7c07c3f8c fix the problems I ran into so it's no longer required.

I'm using the [pnp-webpack-plugin](https://www.npmjs.com/package/pnp-webpack-plugin) to tell Webpack that:

- it should lookup entry and output in the current working directory (where the `webpack` command is executed, meaning the consumer of `@wordpress/scripts`).
- it should lookup loaders in `@wordpress/scripts` and its dependencies, not the consumer.

It seems that `webpack.context` is shared between the entry points and the loaders, so I haven't found any other way to tell Webpack what's the root path it should use to resolve relative paths for entry and loaders.

Reply:
Had to fix up some side-effects of the later changes: afde21c and 805bea7.

Reply:
I've figured we don't actually need the PnpWebpackPlugin.